physionet-django/user/views.py

```diff
@@ -58,14 +60,18 @@
 
 @method_decorator(allow_post_during_maintenance, 'dispatch')
 class LoginView(auth_views.LoginView):
-    template_name = 'user/login.html'
-    extra_context = {
-        'enable_sso': settings.ENABLE_SSO,
-        'sso_login_button_text': settings.SSO_LOGIN_BUTTON_TEXT,
-    }
+    template_name = 'sso/login.html' if settings.ENABLE_SSO else 'user/login.html'
     authentication_form = forms.LoginForm
     redirect_authenticated_user = True
 
+    if settings.ENABLE_SSO:
+        def get_context_data(self, *args, **kwargs):
+            return {
+                'enable_sso': settings.ENABLE_SSO,
+                'sso_login_button_text': settings.SSO_LOGIN_BUTTON_TEXT,
+                'sections': Section.objects.filter(page=Page.LOGIN_INST) if settings.ENABLE_SSO else None
```

Comment:
You already checked whether `settings.ENABLE_SSO` is true a few lines above this one. I'd also name it something different, e.g. `login_instructions.`